.env.testnet

```diff
@@ -0,0 +1,12 @@
+# TestNet env file
+
+# TODO: Below variabls are of privatenet. Update it with testnet env variables
+VUE_APP_NETWORK_NAME = "testnet"
+VUE_APP_BACKEND_URL = "http://localhost:4000/api"
+VUE_APP_ALGOSDK_TOKEN = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+VUE_APP_ALGOSDK_URL = "http://localhost"
```

Comment:
these are used in all env files, is there a way to declare it once and use it in all other files, since if we ever change the endpoint we need to update all the files

Reply:
It won't be unique when SigmaDAO is deployed on the network.